test/futbol_data_test.rb

```diff
@@ -0,0 +1,26 @@
+require "minitest/autorun"
+require "minitest/pride"
+require "./lib/futbol_data"
+require 'mocha/minitest'
+
+class FutbolDataTest < Minitest::Test
+
+  def test_create_chosen_data_set_for_team
+    futbol = FutbolData.new("team")
+    futbol.stubs(:teams).returns("team data array")
```

Comment:
Nice job using mocks and stubs

Reply:
Again, a fun learn and share out! I can't believe I got stuck on this for so long just because I forgot to require the mocha/minitest!!